glide.yaml

```diff
@@ -1,64 +1,34 @@
 package: github.com/openshift/cluster-kube-apiserver-operator
 import:
 - package: k8s.io/apimachinery
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/api
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/apiserver
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/client-go
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/code-generator
-  version: kubernetes-1.12.5
+  version: kubernetes-1.13.4
 - package: k8s.io/apiextensions-apiserver
   repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
   version: 36be1826c980fa7ca82d91645e06e3157f159322
```

Comment:
this one is probably broken now.

Reply:
It is. We need crd publishing first in origin before we can proceed here.